internal/db/storage/blob/s3/reader/reader.go

```diff
@@ -26,34 +26,39 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
-	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/vdaas/vald/internal/backoff"
+	ctxio "github.com/vdaas/vald/internal/db/storage/blob/s3/reader/io"
+	"github.com/vdaas/vald/internal/db/storage/blob/s3/sdk/s3"
+	"github.com/vdaas/vald/internal/db/storage/blob/s3/sdk/s3/s3iface"
 	"github.com/vdaas/vald/internal/errgroup"
 	"github.com/vdaas/vald/internal/errors"
-	ctxio "github.com/vdaas/vald/internal/io"
 	"github.com/vdaas/vald/internal/log"
 	"github.com/vdaas/vald/internal/safety"
 )
 
 type reader struct {
 	eg      errgroup.Group
-	service *s3.S3
+	service s3iface.S3API
```

Comment:
why the package name changed?

Reply:
I wrote the this reason.
could you please read pr description?

Reply:
sorry I am not sure should we use this ?https://github.com/vdaas/vald/blob/d887e713544099356e59942429f3dc8d1378916f/internal/db/storage/blob/s3/sdk/s3/s3.go#L24